File: Answers/05_if/e02_textType.cs
using System;

class Program
{
    static void Main()
    {
        string text = Console.ReadLine();

        if (text.All(char.IsLower))
        {
            Console.WriteLine("Text je malými písmenami.");
        }
        else if (text.All(char.IsUpper))
        {
            Console.WriteLine("Text je veľkými písmenami.");
        }
        else if (text.All(char.IsDigit))
        {
            Console.WriteLine("Text je číslo.");
        }
        else
        {
            Console.WriteLine("Text nie je ani malými písmenami, veľkými písmenami, ani číslo.");
        }
    }
}

File: Answers/05_if/e02_textType.py
text = input("Zadajte text: ")

if text.islower():
    print("Text je malými písmenami.")
elif text.isupper():
    print("Text je veľkými písmenami.")
elif text.isnumeric():
    print("Text je číslo.")
else:
    print("Text nie je ani malými písmenami, veľkými písmenami, ani číslo.")